packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/node/node.ts

```diff
@@ -18,14 +18,18 @@ import type {
 
 const { t } = createI18n<typeof english>('en-us', english)
 
-function getNodeDescription(type: NodeType): string {
+export function getNodeDescription(type: NodeType): string {
   return t(`plugins.free-form.datakit.flow_editor.node_types.${type}.description`)
 }
 
 function getNodeSummary(type: UserNodeType): string {
   return t(`plugins.free-form.datakit.flow_editor.node_types.${type}.summary`)
 }
 
+export function getNodeName(type: NodeType): string {
+  return t(`plugins.free-form.datakit.flow_editor.node_types.${type}.name`)
+}
```

Comment:
What about change this to `getNodeTypeName`? (Also `getNodeTypeDescription` and `getNodeTypeSummary`)